guides/common/modules/con_prerequisites-for-configuring-project-with-quarkus-keycloak-authentication.adoc

```diff
@@ -0,0 +1,23 @@
+[id="prerequisites-for-configuring-{project-context}-with-keycloak-authentication_{context}"]
+= Prerequisites for configuring {Project} with {Keycloak} authentication
+
+Before configuring {Project} with {Keycloak} external authentication, ensure that you meet the following requirements:
+
+* A working installation of {Keycloak} server that uses HTTPS instead of HTTP.
+* A {Keycloak} account with admin privileges.
+* A realm for {Project} user accounts created in {Keycloak}.
+* If the certificates or the CA are self-signed, ensure that they are added to the end-user certificate trust store.
+* Users imported or added to {Keycloak}.
++
+If you have an existing user database configured such as LDAP or Kerberos, you can import users from it by configuring user federation.
+ifndef::orcharhino[]
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_build_of_keycloak/24.0/html/server_administration_guide/user-storage-federation[User Storage Federation] in the _Red{nbsp}Hat build of {Keycloak} Administration Guide_.
```

Comment:
We can use the `RHDocsBaseURL` attribute, but it appears this isn't done consistently throughout our codebase so not blocking IMHO.
```suggestion
For more information, see {RHDocsBaseURL}red_hat_build_of_keycloak/24.0/html/server_administration_guide/user-storage-federation[User Storage Federation] in the _Red{nbsp}Hat build of {Keycloak} Administration Guide_.
```

Reply:
Changed it for all URIs, might be worth a PR to change it for all at once for the rest 🙂